stable/spinnaker/values.yaml

```diff
@@ -104,6 +104,11 @@ redis:
   nodeSelector: {}
   cluster:
     enabled: false
+# Change this to true if youd like to use external redis
+  external: false
```

Comment:
usually when we add support for external alternatives we would have `redis.enabled: true` and then we'd update the requirements.yaml to determine if it should actually include the redis chart ( see the minio conditional right below it https://github.com/helm/charts/blob/master/stable/spinnaker/requirements.yaml#L8 ).

I'd like to see you use that same pattern.

Reply:
Yup. Thanks for the change request. I'll update the redis value structure similar to the one used in minio, and will also update the requirements file.